src/index.html

```diff
@@ -779,8 +1000,8 @@ <h2 class="controls-title">Duration</h2>
           uv.on("openseadragonExtension.doubleClick", function(e) {
             annotations.push({
               target: e.target,
-              bodyValue: String(annotations.length + 1),
-            });
+                bodyValue: String(annotations.length + 1),
+              });

```

Comment:
Reverting unnecessary extra indentation:

```suggestion
              bodyValue: String(annotations.length + 1),
            });

```